.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+tes.md
```

Comment:
- [ ] Please make sure that the test.md is ignored by git and not present in your GitHub repository.
![image](https://user-images.githubusercontent.com/74744787/157318106-fb833c91-6ab4-4a7f-9254-db6715baefac.png)
![image](https://user-images.githubusercontent.com/74744787/157318148-97fe5901-c60f-4eb5-a11a-462207f1936d.png)